### PR TITLE
Branch implement update command

### PR DESCRIPTION
**Pull request by yuhaochua**
@Bryan-BC I have written the UpdateCommand class and it will be expecting a successMessage string to be returned by EquipmentManager.updateEquipment, something like "updated serial number SM57-1, updated details are..."

I have also done the exception handling for wrong enumtype and input for cost during the adding of a new equipment. But I will have to leave the exception handling of these during updating to you because I will be passing over the ArrayList of Pair<String, String>

### CODE REVIEW

**Review by shunyao643 — APPROVED** [review]
Looks good to me, cheers for fixing Parser!

File: src/main/java/seedu/command/AddCommand.java [hunk 1]
```diff
@@ -37,15 +48,14 @@ public CommandResult execute(){
...
         String serialNumber = userInput.get(1);
-        EquipmentType type = userInput.get(2);
+        EquipmentType type = EquipmentType.valueOf(userInput.get(2));
         double cost = Double.parseDouble(userInput.get(3));
         String purchasedFrom = userInput.get(4);
         String purchasedDate = userInput.get(5);
-        Equipment equipment = new Equipment(equipmentName, serialNumber, type, cost, purchasedFrom, purchasedDate);
 
-        equipmentInventory.addEquipment(equipment);
+        equipmentManager.addEquipment(equipmentName, serialNumber, type, cost, purchasedFrom, purchasedDate);
```
Comment (Bryan-BC) [inline src/main/java/seedu/command/AddCommand.java:59]: Would it be okay if we have the same method name addEquipment?

File: src/main/java/seedu/equipment/Equipment.java [hunk 2]
```diff
@@ -64,4 +66,17 @@ public String getPurchasedDate() {
     public void setPurchasedDate(String purchasedDate) {
         this.purchasedDate = purchasedDate;
     }
+
+    @Override
```
Comment (Bryan-BC) [inline src/main/java/seedu/equipment/Equipment.java:70]: Could you tell me what these two methods do?

**Review by Bryan-BC — APPROVED** [review]
LGTM just need some clarification on some things as stated in the comments.

**Pull request merged**